.travis.yml

```diff
@@ -19,14 +19,14 @@ env:
         # Make sure beaver is in the PATH
         - PATH="$(git config -f .gitmodules submodule.beaver.path)/bin:$PATH"
     matrix:
-        - DMD=1.081.1 DIST=xenial F=production AFTER_SCRIPT=1
+        - DMD=1.081.1 DIST=xenial F=production COVERAGE=1 AFTER_SCRIPT=1
```

Comment:
Which coverage report was used before ? Because it was always generated, but I assume `AFTER_SCRIPT` is what was sending it to Codecov ?